EarTrumpet/App.xaml.cs

```diff
@@ -247,5 +249,44 @@ private SettingsCategoryViewModel CreateAddonSettingsPage(IEarTrumpetAddonSettin
         }
 
         private Window CreateMixerExperience() => new FullWindow { DataContext = new FullWindowViewModel(CollectionViewModel) };
+
+
+        private void AbsoluteVolumeIncrement()
+        {
+            foreach (var device in CollectionViewModel.AllDevices)
+            {
+                // if device is muted but not with an absolute control, do not unmute
+                if (device.IsMuted && !device.absMuted)
+                {
+                    continue;
+                }
+
+                // in any case this device is not abs muted anymore
+                device.absMuted = false;
+                device.IncrementVolume(2);
+            }
+        }
```

Comment:
If `AllDevices` is an Enumerable, we should be able to write:

            foreach (var device in CollectionViewModel.AllDevices.Where(d => !d.IsMuted || d.absMuted))
            {
                // in any case this device is not abs muted anymore
                device.absMuted = false;
                device.IncrementVolume(2);
            }
            
Note that `!(a && !b)` is `!a || b`